SourceryFramework/Sources/Parsing/Composer.swift

```diff
@@ -76,11 +74,15 @@ public struct Composer {
             unique[current.name] = current
         }
 
+        let resolutionStart = CACurrentMediaTime()
+
         let resolveType = { (typeName: TypeName, containingType: Type?) -> Type? in
             return self.resolveType(typeName: typeName, containingType: containingType, unique: unique, modules: modules, typealiases: typealiases)
         }
 
-        for (_, type) in unique {
+        let array = Array(unique.values)
+        DispatchQueue.concurrentPerform(iterations: array.count) { (counter) in
```

Comment:
TIL

Reply:
🤯

Reply:
we have our own parallel map in the main Sourcery target for parallel `flatMap` / `map` but for just multi-threaded apply this is good.